### PR TITLE
Adds RPED to Engieborg

### PR DESCRIPTION
**Pull request by Oyu07**
## What Does This PR Do

Changes mechfab design to use SECONDS

Adds RPED (Rapid Part Exchange Device) as an upgrade for Engineering Borg 

Research
- Materials 3
- Engineering 4

Materials
- Metal Sheet 20
- Glass Sheet 7.5

No bluespace variant. Borgo doesnt really need it, this is meant as a QoL and utility item, not to replace RnD's job to upgrade stuff

## Why It's Good For The Game

Engieborg is behind in terms of building machines. If an engieborg wants to build a machine, lets say a borg charger in medbay, they need to carry a whole closet just for 3 items. 

This PR adds RPED. which allows engieborgs to build machines without the tedious locker.

## Images of changes

dreamseeker_54xde3I5ob

## Testing
1. Compiles
2. Max RND
3. Fabricate via Mech Fabricator
4. Install on engie borg
5. Works just like a normal RPED

## Changelog
:cl:
add: Engieborg Rapid Part Exchange Device upgrade
/:cl:

### CODE REVIEW

File: code/game/objects/items/robot/robot_upgrades.dm [hunk 1]
```diff
@@ -461,3 +461,12 @@
 		return
 	R.module.remove_item_from_lists(/obj/item/rcd) // So emagging them in the future won't grant another RCD.
 	..()
+
+/obj/item/borg/upgrade/rped
+	name = "Rapid Part Exchange Device upgrade"
+	desc = "A modified Rapid Part Exchange Device designed to be used by engineering robots"
```
Comment (Henri215) [inline code/game/objects/items/robot/robot_upgrades.dm:467]:
```suggestion
	desc = "A modified Rapid Part Exchange Device designed to be used by engineering robots."
```

File: code/game/objects/items/robot/robot_upgrades.dm [hunk 2]
```diff
@@ -461,3 +461,12 @@
...
 	R.module.remove_item_from_lists(/obj/item/rcd) // So emagging them in the future won't grant another RCD.
 	..()
+
+/obj/item/borg/upgrade/rped
+	name = "Rapid Part Exchange Device upgrade"
+	desc = "A modified Rapid Part Exchange Device designed to be used by engineering robots."
+	icon_state = "cyborg_upgrade5"
+	require_module = TRUE
+	module_type = /obj/item/robot_module/engineering
+	items_to_add = list(/obj/item/storage/part_replacer)
```
Comment (Henri215) [inline code/game/objects/items/robot/robot_upgrades.dm:471]: This wont need a cyborg version right?

Comment (Oyu07) [reply code/game/objects/items/robot/robot_upgrades.dm:471]: Nope

There is literally no change, just the RPED itself but on borgs.

File: code/modules/research/designs/mechfabricator_designs.dm [hunk 3]
```diff
@@ -1138,14 +1138,24 @@
...
+
 /datum/design/borg_upgrade_rcd
 	name = "Cyborg Upgrade (Rapid Construction Device)"
 	id = "borg_upgrade_RCD"
 	build_type = MECHFAB
 	build_path = /obj/item/borg/upgrade/rcd
 	req_tech = list("materials" = 6, "engineering" = 5, "powerstorage" = 5)
-	materials = list(MAT_METAL=30000, MAT_GLASS=15000,)
+	materials = list(MAT_METAL = 30000, MAT_GLASS = 15000)
+	construction_time = 120
```
Comment (Sirryan2002) [inline code/modules/research/designs/mechfabricator_designs.dm:1149]:
```suggestion
	construction_time = 12 SECONDS
```

**Review by Sirryan2002 — APPROVED** [review]

**Comment by Oyu07** [comment]
Merge conflicted from the organization PR.

Will de conflict tomorrow

File: code/game/objects/items/robot/robot_upgrades.dm [hunk 4]
```diff
@@ -366,6 +366,14 @@
...
 	..()
 
+/obj/item/borg/upgrade/rped
+	name = "Rapid Part Exchange Device upgrade"
+	desc = "A modified Rapid Part Exchange Device designed to be used by engineering robots."
+	icon_state = "cyborg_upgrade5"
+	require_module = TRUE
+	module_type = /obj/item/robot_module/engineering
+	items_to_add = list(/obj/item/storage/part_replacer)
+
```
Comment (DGamerL) [inline code/game/objects/items/robot/robot_upgrades.dm:376]:
```suggestion
```

**Review by DGamerL — APPROVED** [review]

**Review by Henri215 — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**